test/util/util.go

```diff
@@ -0,0 +1,111 @@
+package util
```

Comment:
Can we rename the package as "metallb" and the exposed functions accordingly?
From the calling site it will be metallb.Get() / metallb.Delete / etc which is more readable.

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
done

Reply:
Can you also change the name of the file?

Reply:
done